pages/common/gitlab-ctl.md

```diff
@@ -0,0 +1,28 @@
+# gitlab-ctl
+
+> A CLI tool for managing the GitLab omnibus.
```

Comment:
Perhaps this will read better?
```suggestion
> CLI tool for managing the GitLab omnibus.
```

Reply:
@navarroaxel asked me to do the exact opposite in https://github.com/tldr-pages/tldr/pull/4758#discussion_r507657475 :stuck_out_tongue:

Reply:
There are 13 pages in tldr using `A CLI...` :man_shrugging: Maybe we should fix them. @prudloff-insite please follow the suggestion from @sbrl.